source/chapters/user_interface.rst

```diff
@@ -79,13 +87,25 @@ Track Information Display
             to the speed you are tapping, go to :ref:`control-keyboard` for more
             information.
 
-**Time Elapsed/Remaining/Both**
-  By default it displays the total elapsed time in the track up to the
-  millisecond. Clicking on the display switches to the *Time Remaining* view,
-  which indicates how much time is left before the track reaches the end.
-  Clicking the display again, both the elapsed and remaining time are shown
-  simultaneously in the format *elapsed time / remaining time*.
-  You can change the default in :menuselection:`Preferences --> Decks --> Deck options`.
+**Key**
+  .. figure:: ../_static/2.2/key-options.png
```

Comment:
you can use this picture (intermediate fix sowe don't need to change the picture late ron)

![Screenshot_2020-10-29_18-46-41](https://user-images.githubusercontent.com/5934199/97612048-34f56700-1a17-11eb-966d-aa456d76cd8d.png)

Reply:
Okay.